reflex/testing.py

```diff
@@ -159,6 +159,7 @@ def _initialize_app(self):
             reflex.config.get_config(reload=True)
             self.app_module = reflex.utils.prerequisites.get_app()
         self.app_instance = self.app_module.app
+        self.app_instance._compile()
```

Comment:
@masenf this doesn't seem to be correct or enough, any idea what I should do instead?

Reply:
nest these calls under that `chdir` context manager above; all of the compiler code uses relative file paths, so if you're not in the app directory, then the files get compiled to the working dir

Reply:
ah that makes sense. Thanks!